app/src/main/java/org/oppia/app/topic/overview/TopicOverviewFragmentPresenter.kt

```diff
@@ -35,7 +35,7 @@ class TopicOverviewFragmentPresenter @Inject constructor(
   private val topicOverviewViewModel = getTopicOverviewViewModel()
   private lateinit var topicId: String
   private val htmlParser: HtmlParser by lazy {
-    htmlParserFactory.create(/* entityType= */ "topic", topicId)
+    htmlParserFactory.create(/* entityType= */"topic", topicId, /* imageCenterAlign= */ true)
```

Comment:
@veena14cs Is this correct?

Reply:
I think this needs to be in center only.

Reply:
This is correct if here we are only showing content.